src/library_signals.js

```diff
@@ -1,37 +1,40 @@
 // 'use strict'
-var funs = {
-  _sigalrm_handler: 0,
+var funs = 
+{
+	signal__deps: ['_sig_handler_'],
 
-  signal__deps: ['_sigalrm_handler'],
-  signal: function(sig, func) {
-    if (sig == 14 /*SIGALRM*/) {
-      __sigalrm_handler = func;
-    } else {
-#if ASSERTIONS
-      Module.printErr('Calling stub instead of signal()');
-#endif
-    }
-    return 0;
-  },
-  sigemptyset: function(set) {
-    {{{ makeSetValue('set', '0', '0', 'i32') }}};
-    return 0;
-  },
-  sigfillset: function(set) {
-    {{{ makeSetValue('set', '0', '-1>>>0', 'i32') }}};
-    return 0;
-  },
-  sigaddset: function(set, signum) {
-    {{{ makeSetValue('set', '0', makeGetValue('set', '0', 'i32') + '| (1 << (signum-1))', 'i32') }}};
-    return 0;
-  },
-  sigdelset: function(set, signum) {
-    {{{ makeSetValue('set', '0', makeGetValue('set', '0', 'i32') + '& (~(1 << (signum-1)))', 'i32') }}};
-    return 0;
-  },
-  sigismember: function(set, signum) {
-    return {{{ makeGetValue('set', '0', 'i32') }}} & (1 << (signum-1));
-  },
+	signal: function(sig, func) 
```

Comment:
i my emscripten source, i have done others changes
to library, but not testing at the moment
you're right. 
this function have no changes.